packages/terra-data-grid/tests/jest/__snapshots__/FlowsheetDataGrid.test.jsx.snap

```diff
@@ -0,0 +1,99 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`FlowsheetDataGrid renders the row header column as pinned and remaining columns as overflow and all columns as not resizable 1`] = `
+<div
+  className="flowsheet-data-grid-container"
+>
+  <InjectIntl(DataGrid)
+    ariaLabel="Test Flowsheet Data Grid"
+    columnHeaderHeight="2.5rem"
+    defaultColumnWidth={200}
+    id="test-terra-flowsheet-data-grid"
+    overflowColumns={
+      Array [
+        Object {
+          "displayName": "March 16",
+          "id": "Column-1",
+          "isResizable": false,
```

Comment:
We should test not passing a value for isResizable as well as passing a value of true.

Reply:
Yes, that is what I tried to do by passing 'true' for col[0] and col[2] and nothing for col[1] in the test data. The end result in this snapshot is that they are all 'false' as we would expect.